extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -125,6 +128,32 @@ class AmpStickyAd extends AMP.BaseElement {
       });
     }
   }
+
+  /**
+   * The function that add a close button to sticky ad
+   * @private
+   */
+  addCloseButton_() {
+    const closeButton = this.getWin().document.createElement('div');
+    closeButton.classList.add('-amp-sticky-ad-close-button');
+    closeButton.setAttribute('aria-label','Close');
```

Comment:
We don't have yet any i18n mechanism in our JS code. Should we add something for this now? Also, since assistive tech should scroll right through main content - is it really important to provide y11a for close button here?